game/plugins/src/main/kotlin/gg/rsmod/plugins/api/ext/StringExt.kt

```diff
@@ -31,4 +33,31 @@ fun String.withPluralSuffix(string: String, count: Int) : String {
  */
 fun String.prefixAn() : String {
     return if (vowels.indexOf(Character.toLowerCase(this[0])) != -1) "an $this" else "a $this"
+}
+
+fun String.splitForDialogue() : Array<String> {
```

Comment:
Oooo! is this a method to split dialogue based on a max number of characters? Very cool indeed.